rpcclient/wallet.go

```diff
@@ -1015,7 +1036,7 @@ func (c *Client) GetAccountAddressAsync(account string) FutureGetAccountAddressR
 
 // GetAccountAddress returns the current Bitcoin address for receiving payments
 // to the specified account.
-func (c *Client) GetAccountAddress(account string) (btcutil.Address, error) {
+func (c *Client) GetAccountAddress(account string) (string, error) {
```

Comment:
Instead, the chain params should be passed in.

Reply:
Thanks, good idea! Preparing an improving commit.